javascript/builtins/Intl.json

```
@@ -109,7 +109,7 @@
                 "version_added": false
               },
               "nodejs": {
-                "version_added": false
+                "version_added": "18.4.0"
```

Based on https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V18.md#v8-101, it looks like this was actually added in 18.0 rather than 18.4:

```suggestion
                "version_added": "18.0.0"
```